packages/rlc-common/CHANGELOG.md

```diff
@@ -43,7 +43,7 @@
 - [Feature] Support additional properties full cases in RLC. Please refer to [#2445](https://github.com/Azure/autorest.typescript/pull/2445).
 - [Feature] Support the change for union and enums in modular and RLC. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
 - [Feature] Upgrade typespec compiler to 0.55.0. Please refer to [#2380](https://github.com/Azure/autorest.typescript/pull/2380).
-- [Feature] Bump typescript version to typescript@~5.4.5. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
+- [Feature] Bump typescript version to typescript@~5.5.3. Please refer to [#2442](https://github.com/Azure/autorest.typescript/pull/2442).
```

Comment:
feel like we don't need to modify the changelog in the past?

Reply:
yes you are right! I was replacing strings and forgot to undo this file

Reply:
reverted.